rerun_py/rerun_sdk/rerun/__init__.py

```diff
@@ -60,42 +97,6 @@
     "TranslationAndMat3x3",
     "TranslationRotationScale3D",
     "ViewCoordinates",
-    "archetypes",
-    "bindings",
-    "blueprint",
-    "components",
-    "connect",
-    "datatypes",
-    "disable_timeline",
-    "disconnect",
-    "escape_entity_path_part",
-    "experimental",
-    "get_application_id",
-    "get_data_recording",
-    "get_global_data_recording",
-    "get_recording_id",
-    "get_thread_local_data_recording",
-    "is_enabled",
-    "log",
-    "log_components",
-    "log_file_from_contents",
-    "log_file_from_path",
-    "memory_recording",
-    "new_entity_path",
-    "notebook_show",
-    "reset_time",
-    "save",
-    "script_add_args",
-    "script_setup",
-    "script_teardown",
-    "serve",
-    "set_global_data_recording",
-    "set_thread_local_data_recording",
-    "set_time_nanos",
-    "set_time_seconds",
-    "set_time_sequence",
-    "send_blueprint",
-    "spawn",
```

Comment:
I wonder who has the type-sensitive sorter. I guess we'll find out in a PR soon enough

Reply:
Nope, we won't 😆
* https://github.com/rerun-io/rerun/pull/6117